### PR TITLE
Uploading test results

### PR DESCRIPTION
**Pull request by shuheiktgw**
Uploading test results for dynamic reordering.

# Todos
- [ ] Receive build number through the environment variable and the command line argument both

### CODE REVIEW

File: launchable/uploader.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    def _get_success_interval(cls):
+        return os.getenv(cls.SUCCESS_REPORT_INTERVAL_KEY) or cls.DEFAULT_SUCCESS_REPORT_INTERVAL
+
+    @classmethod
+    def _get_failure_interval(cls):
+        return os.getenv(cls.FAILURE_REPORT_INTERVAL_KEY) or cls.DEFAULT_FAILURE_REPORT_INTERVAL
+
+
+class Uploader:
+    MAX_BATCH_SIZE = 500
```
Comment (shuheiktgw) [inline launchable/uploader.py:30]: This is not necessary

**Comment by ninjinkun** [comment]
I tried to refactor this PR with `asyncio` for 3 hours. However, I finally agreed original thread code was better. I removed my code. Now, I understand this PR deeply, and ready to approve this PR.

**Review by ninjinkun — APPROVED** [review]

**Comment by ninjinkun** [comment]
@shuheiktgw Is this ready to merge?

**Comment by shuheiktgw** [comment]
I need to modify the test result format so give me a little bit more time! 🙏

**Comment by ninjinkun** [comment]
@shuheiktgw Thank you!

**Comment by shuheiktgw** [comment]
Wait!

**Comment by shuheiktgw** [comment]
I found one stupid mistake so give me a moment...

**Comment by shuheiktgw** [comment]
And fixed it

**Comment by ninjinkun** [comment]
@shuheiktgw Thank you for fixing this conflict. 
By the way, please stop force push from the next time. I can't follow what you changed.

**Comment by shuheiktgw** [comment]
Whoops sorry, I should have used squash and merge instead!

**Pull request merged**